### PR TITLE
Certification fixes

### PR DESCRIPTION
**Pull request by nCastle1**
Most of the changes to fix issues found in certification are in. Left to do:

* Review for consistency (were changes applied consistently to samples across all platforms?)
* ~Update screenshots - several samples had UI or data changes~
* Review comments - are comments consistent with actual sample
* Re-certify samples on all platforms
* Make sure nothing unwanted slipped in (e.g. changes to project files, leftover changes from testing, etc.)

### CODE REVIEW

**Comment by nCastle1** [comment]
@dkemlage Do you have some time to take a look at this? I've made several changes to resolve feedback from certification. It's not quite done (you'll notice missing screenshots) but given the compressed timeline I'd like to get started early if you have the availability. Thanks :)

File: src/Forms/Shared/Samples/Symbology/SimpleRenderers/SimpleRenderers.xaml.cs [hunk 1]
```diff
@@ -49,6 +46,12 @@ private void Initialize()
             // Use the two points farthest apart to create an envelope
             Envelope initialEnvelope = new Envelope(oldFaithfullPoint, plumeGeyserPoint);
 
+            // Use the envelope to define the map views visible area (include some padding around the extent)
+            myMap.InitialViewpoint = new Viewpoint(initialEnvelope, 100);
```
Comment (fragmer) [inline src/Forms/Shared/Samples/Symbology/SimpleRenderers/SimpleRenderers.xaml.cs]: For some reason the MapView starts off rotated. This does NOT affect usability of the sample — no changes needed.

I'll have to investigate why this happens on XF-UWP and not on native WPF/UWP. Might be related to that viewpoint bug I mentioned on RasterLayer...

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Map/ManageBookmarks/ManageBookmarks.xaml.cs [hunk 2]
```diff
@@ -82,6 +84,12 @@ private void Initialize()
             BorderAddBookmark.Visibility = System.Windows.Visibility.Hidden;
         }
 
+        private void LocationDisplay_LocationChanged(object sender, Esri.ArcGISRuntime.Location.Location e)
```
Comment (fragmer) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Map/ManageBookmarks/ManageBookmarks.xaml.cs]: Did you mean to do something with this event (other than unsubscribing)?

File: src/Android/Xamarin.Android/Samples/Search/FindPlace/FindPlace.cs [hunk 3]
```diff
@@ -267,8 +283,8 @@ private void CreateLayout()
             pinSymbol.Height = 60;
             // The image is a pin; offset the image so that the pinpoint
             //     is on the point rather than the image's true center
-            pinSymbol.OffsetX = pinSymbol.Width / 2;
-            pinSymbol.OffsetY = pinSymbol.Height / 2;
+            pinSymbol.LeaderOffsetX = 30;
+            pinSymbol.OffsetY = 14;
```
Comment (fragmer) [inline src/Android/Xamarin.Android/Samples/Search/FindPlace/FindPlace.cs]: Apply same fix to FindAddress sample also (on all platforms)

File: src/Android/Xamarin.Android/Samples/Layers/RasterLayerFile/RasterLayerFile.cs [hunk 4]
```diff
@@ -77,7 +77,7 @@ private void CreateLayout()
             Viewpoint myFullRasterExtent = new Viewpoint(myRasterLayer.FullExtent);
 
             // Zoom to the extent
-            _myMapView.Map.InitialViewpoint = myFullRasterExtent;
+            _myMapView.SetViewpoint(myFullRasterExtent);
```
Comment (fragmer) [inline src/Android/Xamarin.Android/Samples/Layers/RasterLayerFile/RasterLayerFile.cs]: Turns out this doesn't work on Android either. I'm not sure if the [suggested workarounds for XF](https://github.com/Esri/arcgis-runtime-samples-dotnet/pull/364#discussion_r154547385) will work here — you'll have to give it a try.

File: src/UWP/ArcGISRuntime.UWP.Samples/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.xaml [hunk 5]
```diff
@@ -10,7 +10,14 @@
...
             <Grid>
-                <ListBox x:Name="MyDisplayList" SelectionMode="Multiple" SelectionChanged="MyDisplayList_SelectionChanged_1" />
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                </Grid.RowDefinitions>
+                <TextBlock Text="Select a WMS layer from the list for display:" Grid.Row="0" />
+                <ListBox x:Name="MyDisplayList" SelectionMode="Multiple" SelectionChanged="MyDisplayList_SelectionChanged_1" Grid.Row="1" />
+                <TextBlock Foreground="Red" Text="Note: above list does not show layer hierarchy" Grid.Row="2" />
```
[the remark this answers is not in the document]
Comment (nCastle1) [reply src/UWP/ArcGISRuntime.UWP.Samples/Samples/Layers/WmsServiceCatalog/WmsServiceCatalog.xaml]: I made an issue to fix the UWP UIs. I didn't think I'd have time to fix all of the affected samples before release.

File: src/UWP/ArcGISRuntime.UWP.Viewer/MainPage.xaml [hunk 6]
```diff
@@ -190,10 +190,6 @@
                        Text="{Binding ElementName=categories, Path=SelectedItem.Name}"
                        Foreground="{StaticResource PrimaryColor}"
                        FontSize="24"></TextBlock>
-            <AutoSuggestBox x:Name="SearchBox" RelativePanel.AlignRightWithPanel="True"
```
[the remark this answers is not in the document]
Comment (nCastle1) [reply src/UWP/ArcGISRuntime.UWP.Viewer/MainPage.xaml]: I did intend to hide it completely, I must have overlooked on mobile, thanks for the catch.

File: src/Forms/Shared/Samples/Layers/RasterLayerFile/RasterLayerFile.xaml.cs [hunk 7]
```diff
@@ -51,7 +51,7 @@ public RasterLayerFile()
 			Viewpoint myFullRasterExtent = new Viewpoint(myRasterLayer.FullExtent);
 
             // Zoom to the extent
-            MyMapView.Map.InitialViewpoint = myFullRasterExtent;
+            MyMapView.SetViewpoint(myFullRasterExtent);
```
[the remark this answers is not in the document]
Comment (fragmer) [reply src/Forms/Shared/Samples/Layers/RasterLayerFile/RasterLayerFile.xaml.cs]: I retested this one and navigating to the right area is still somewhat unpredictable. Might be worth investigating this further in a separate issue.

**Pull request merged**